Include/internal/pycore_frame.h

```diff
@@ -101,7 +101,7 @@ void _PyFrame_Copy(_PyInterpreterFrame *src, _PyInterpreterFrame *dest);
    when frame is linked into the frame stack.
  */
 static inline void
-_PyFrame_InitializeSpecials(
+_PyFrame_InitializeHeader(
```

Comment:
If you initialize the locals, then this initializes the whole frame. Maybe call it `_PyFrame_Initialize`?